helix-tui/src/text.rs

```diff
@@ -428,6 +428,19 @@ impl<'a> From<Vec<Spans<'a>>> for Text<'a> {
     }
 }
 
+impl<'a> From<Text<'a>> for String {
+    fn from(text: Text<'a>) -> String {
+        let lines: Vec<String> = text.lines.iter().map(String::from).collect();
+        lines.join("\n")
```

Comment:
Would collecting into `String` directly be faster?

Reply:
Done in b28f684d1d.